types/formidable/index.d.ts

```diff
@@ -34,7 +34,7 @@ export interface Fields {
 }
 
 export interface Files {
-    [key: string]: File; // | File[];
+    [key: string]: File | File[];	// is an array when multiples is true
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/formidable/index.d.ts;
  Line: 37;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.